spec/site_prism/page_objects/pages/cases/extend_sar_deadline_page.rb

```diff
@@ -0,0 +1,30 @@
+module PageObjects
+  module Pages
+    module Cases
+      class ExtendSARDeadlinePage < SitePrism::Page
+        set_url '/cases/{id}/extend_sar_deadline'
+
+        section :primary_navigation,
+          PageObjects::Sections::PrimaryNavigationSection,
+          '.global-nav'
+
+        section :page_heading,
+          PageObjects::Sections::PageHeadingSection,
+          '.page-heading'
+
+        element :copy, '.action-copy'
+
+        # Form
```

Comment:
Is this just a note? I think if it's necessary to put it in then it should be a section?